website/docs/docs/build/semantic-models.md

```diff
@@ -84,7 +84,7 @@ semantic_models:
       - name: transaction_date
         type: time
         type_params:
-          time_granularity: day
+          time_granularity: day # Additional options include hour, week, month, quarter, year, and so on.
```

Comment:
seems weird to exclude other options like second and below if we're going to list so many. do we need this list at all?